egs/audioset/AT/local/generate_audioset_manifest.py

```diff
@@ -0,0 +1,134 @@
+import argparse
+import csv
+import glob
+import logging
+
+import torch
+from lhotse import CutSet, Fbank, FbankConfig, LilcomChunkyWriter
+from lhotse.audio import Recording
+from lhotse.cut import MonoCut
+from lhotse.supervision import SupervisionSegment
+
+from icefall.utils import get_executor
+
+torch.set_num_threads(1)
+torch.set_num_interop_threads(1)
+
+
+def parse_csv(csv_file="downloads/audioset/full_train_asedata_with_duration.csv"):
+
+    mapping = {}
+    with open(csv_file, "r") as fin:
+        reader = csv.reader(fin, delimiter="\t")
+        for i, row in enumerate(reader):
+            if i == 0:
+                continue
+            key = "/".join(row[0].split("/")[-2:])
+            mapping[key] = row[1]
+    return mapping
+
+
+def get_parser():
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+
+    parser.add_argument("--dataset-dir", type=str, default="downloads/audioset")
+
+    parser.add_argument(
+        "--split",
+        type=str,
+        default="balanced",
+        choices=["balanced", "unbalanced", "eval", "eval_all"],
+    )
+
+    parser.add_argument(
+        "--feat-output-dir",
+        type=str,
+        default="data/fbank_audioset",
```

Comment:
https://github.com/k2-fsa/icefall/blob/7a8c9b7f53b35d2b72d2f64f6547c1c367118462/egs/audioset/AT/zipformer/at_datamodule.py#L93

Could you use a consistent default value?

Reply:
Just made them the same default value